pkgs/by-name/se/sequential/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sequential";
+  version = "2.6.0";
+
+  src =
+    let
+      buildDate = "2024-09-07.14.59.00";
+    in
+    fetchurl {
+      url = "https://github.com/chuchusoft/Sequential/releases/download/v${finalAttrs.version}/Sequential.app.${buildDate}.tar.xz";
+      hash = "sha256-tgpzMAHw266UhKo43GIHFCx/SDq/zIJkWz1TPYTeTzI=";
+    };
```

```suggestion
  buildDate = "2024-09-07.14.59.00";

  src = fetchurl {
      url = "https://github.com/chuchusoft/Sequential/releases/download/v${finalAttrs.version}/Sequential.app.${finalAttrs.buildDate}.tar.xz";
      hash = "sha256-tgpzMAHw266UhKo43GIHFCx/SDq/zIJkWz1TPYTeTzI=";
    };
```

to make it overridable